tests/unit/lms/views/lti/basic_launch_test.py

```diff
@@ -182,12 +182,6 @@ def test__show_document(
             sentinel.document_url, assignment_extra=sentinel.assignment_extra
         )
 
-        context.js_config.enable_lti_launch_mode.assert_called_once_with()
-        context.js_config.set_focused_user.assert_not_called()
-        context.js_config.add_document_url.assert_called_once_with(
-            sentinel.document_url
-        )
```

Comment:
Moved this down to better follow the layout of the tested code and to make getting the assignment easier